identity-credential-directive.js

```diff
@@ -0,0 +1,247 @@
+/*!
+ * Identity Credential.
+ *
+ * Copyright (c) 2015 Digital Bazaar, Inc. All rights reserved.
+ *
+ * @author Alex Lamar
+ */
+define(['angular', 'jsonld', 'underscore'], function(angular, jsonld, _) {
+
+'use strict';
+
+/* @ngInject */
+function brIdentityCredential($rootScope, brCredentialLibraryService) {
+  return {
+    restrict: 'E',
+    scope: {
+      doneCallback: '&brCallback',
+      credential: '=credential',
+      selectedCredentials: '=selectedCredentials',
+      allCredentials: '=allCredentials',
+      query: '=query',
+      library: '=?library',
+    },
+    /* jshint multistr: true */
+    templateUrl: requirejs.toUrl(
+      'bedrock-angular-identity-composer/identity-credential.html'),
+    link: Link
+  };
+
+  function Link(scope, element, attrs) {
+    var CONTRACTION_DEFAULT = 2;
+    scope.contractionAmount = CONTRACTION_DEFAULT;
```

Comment:
Is `CONTRACTION_DEFAULT` or `scope.contractionAmount` used anywhere? What do they mean? If unused, remove. If used, perhaps rename or document.